commit 364eb72a3965ddf3291be74562081b1c19813473
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Thu Oct 7 16:11:01 2010 -0700

    Staging: brcm80211: remove MY_INIT_WORK macro
    
    Use the real macro instead.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/linuxver.h
+++ b/drivers/staging/brcm80211/include/linuxver.h
@@ -34,8 +34,6 @@
 
 #include <linux/workqueue.h>
 
-#define	MY_INIT_WORK(_work, _func)	INIT_WORK(_work, _func)
-
 typedef irqreturn_t(*FN_ISR) (int irq, void *dev_id, struct pt_regs * ptregs);
 
 #include <linux/sched.h>
--- a/drivers/staging/brcm80211/sys/wl_mac80211.c
+++ b/drivers/staging/brcm80211/sys/wl_mac80211.c
@@ -831,7 +831,7 @@ static wl_info_t *wl_attach(uint16 vendo
 	}
 
 	/* init tx work queue for wl_start/send pkt; no need to destroy workitem  */
-	MY_INIT_WORK(&wl->txq_task.work, (work_func_t) wl_start_txqwork);
+	INIT_WORK(&wl->txq_task.work, (work_func_t) wl_start_txqwork);
 	wl->txq_task.context = wl;
 #endif				/* WLC_HIGH_ONLY */
 
@@ -1670,7 +1670,7 @@ wl_schedule_task(wl_info_t *wl, void (*f
 		return -ENOMEM;
 	}
 
-	MY_INIT_WORK(&task->work, (work_func_t) fn);
+	INIT_WORK(&task->work, (work_func_t) fn);
 	task->context = context;
 
 	if (!schedule_work(&task->work)) {
